raiden/tests/unit/test_channelstate.py

```diff
@@ -95,15 +105,15 @@ def create_model(balance, merkletree_width=0):
         amount_locked=0,
         balance=balance,
         distributable=balance,
-        next_nonce=1,
+        next_nonce=len(merkletree_leaves) + 1,
         merkletree_leaves=merkletree_leaves,
         contract_balance=balance,
     )
 
     return our_model, privkey
 
 
-def create_channel_from_models(our_model, partner_model):
+def create_channel_from_models(our_model, partner_model, partner_pkey):
```

Comment:
because the factory now creates valid balance proofs we need the partner private key to create the signed balance proof